gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx

```diff
@@ -19,15 +19,15 @@ function NxDropdownActionsExample() {
                 onToggleCollapse={onToggleCollapse}>
       <a href="https://www.google.com/" className="nx-dropdown-link">
         <span>Go somewhere</span>
-        <NxFontAwesomeIcon icon={faExternalLinkAlt}/>
+        <NxFontAwesomeIcon className="nx-icon--colorful" icon={faExternalLinkAlt}/>
```

Comment:
We should probably build this into the `.nx-dropdown-link` styling rather than requiring the user to specify it.  That is, when an icon is within `.nx-dropdown-link` and _does not_ have `.nx-icon--colorful`, we should set its color to the link blue.

Reply:
> We should probably build this into the `.nx-dropdown-link` styling rather than requiring the user to specify it. That is, when an icon is within `.nx-dropdown-link` and _does not_ have `.nx-icon--colorful`, we should set its color to the link blue.

Fixed here: f300e51084a77e4f23ff4be98780714f48b13e5b